src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -1,64 +1,54 @@
 package seedu.address.logic.parser;
 
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ENTRYBOOK_FLOATINGTASK;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FLOATINGTASK;
 
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
-import java.util.stream.Stream;
 
 import seedu.address.commons.core.index.Index;
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.DeleteByFindCommand;
+import seedu.address.logic.commands.DeleteByIndexCommand;
 import seedu.address.logic.commands.DeleteCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
-//@@kevinlamkb A0140633R
 /**
  * Parses input arguments and creates a new DeleteCommand object
  */
 public class DeleteCommandParser {
 
     /**
-     * Parses the given {@code String} of arguments in the context of the
-     * DeleteCommand and returns an DeleteCommand object for execution.
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand and returns an
+     * DeleteCommand object for execution.
      *
      * @throws ParseException
-     * if the user input does not conform the expected format
+     *             if the user input does not conform the expected format
```

Comment:
hmm yes. that's weird, eclipse auto formatting is doing that for me even when the original comments isnt over 110 characters.

anyway, thanks. fixed.